samples/listctrl/listtest.cpp

```diff
@@ -267,6 +268,7 @@ MyFrame::MyFrame(const wxString& title)
     menuList->AppendCheckItem(LIST_TOGGLE_HEADER, "Toggle &header\tCtrl-H");
     menuList->Check(LIST_TOGGLE_HEADER, true);
     menuList->AppendCheckItem(LIST_TOGGLE_BELL, "Toggle &bell on no match");
+    menuList->Append( LIST_CHECKVISIBILITY, "Check if line is visible" );
```

Comment:
```suggestion
    menuList->Append( LIST_CHECKVISIBILITY, "Check if lines 2 and 9 are visible" );
```
Or something else closer to what this menu item actually does.